spec/requests/api/v1/users/update_spec.rb

```diff
@@ -0,0 +1,78 @@
+RSpec.describe 'PUT api/v1/users', type: :request do
+  let(:user) { create(:user) }
```

Comment:
Wait, where is the user being used? Does `#auth_headers` automagically know the user? That's not nice